apps/editor/src/js/ui/popupAddImage.js

```diff
@@ -108,26 +108,30 @@ class PopupAddImage extends LayerPopup {
     this.on('hidden', () => this._resetInputs());
 
     this.on(`change .${CLASS_IMAGE_FILE_INPUT}`, () => {
-      const filename = this._imageFileInput.value.split('\\').pop();
+      const filename = Array.prototype.map
+        .call(this._imageFileInput.files, file => file.name)
+        .join('、');
```

Comment:
Why is it concatenated with `、`?

Reply:
just a symbol to concatenate the names of files. I think it's not necessnary to provide a configuration, so I did that.

Reply:
I think you tried to write a comma(`,`), but the text you used doesn't seem to be a comma.
![charCode](https://user-images.githubusercontent.com/18183560/78756903-8b855000-79b6-11ea-8b87-a170a3b0235b.png)

Reply:
oh sorry. (`、`) is only in chinese, which is also means comma. I will change it to (`,`)